vp8_remb_simulcast.h

```diff
@@ -0,0 +1,43 @@
+#include <glib.h>
+#include "rtp.h"
+#include "mutex.h"
+
+/* This is a helper object that is used to pass publisher related data to the methods of the vp8_remb_simulcast implementation */
```

Comment:
The struct and properties need the `! \brief` prefix to be detected by Doxygen.

Reply:
Adopted